### DISCUSSION TITLE
Infinite push on Android

### DISCUSSION THREAD

**Discussion by freenandes** [Vinzent03/obsidian-git#644] [Problems]
How do you folks solve the infinite push in Android? From the docs i get that i should look into authentication. But the thing is, what should i do there? I already reset the github token and i'm not getting any change on the behavior, it's stuck on infinite push. I can pull though...

**Comment by artemlenkevich** [comment]
> How do you folks solve the infinite push in Android? From the docs i get that i should look into authentication. But the thing is, what should i do there? I already reset the github token and i'm not getting any change on the behavior, it's stuck on infinite push. I can pull though...

Yeah, same happened to me
